.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,105 @@
+# Run benchmark report on pull requests to master.
+# The report is added to the PR as a comment.
```

Comment:
This file is the github action for running benchmarks on pull request, and comparing them with the latest commit on master.